kolibri/core/assets/src/views/AppBar.vue

```diff
@@ -250,7 +250,7 @@
 
   .user-menu-dropdown {
     position: fixed;
-    right: 0;
+    right: 20px;
```

Comment:
Let's try a smaller value like 8px as a start. 

```suggestion
    right: 8px;
```